### PR TITLE
Create data model

### PR DESCRIPTION
**Pull request by OLIPLICHE**
I used migration files to build the schema:

- I created and run the necessary migration files.
- Table and column names match the ERD diagram.

### CODE REVIEW

**Review by mke2111 — CHANGES_REQUESTED** [review]
### Hey @OLIPLICHE, 

**This is a great project you have going on, well-done lad**. 

   
**However, there are a few sections to improve before your project is approved.** 🔴 
<hr>

## Required Changes ♻️

- [ ] In the readme file, add the complete instructions on how to set up on a local computer. Remember to add all the necessary commands such as (migration, package installers)

_Check the inline comments under the review for sections to improve._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: db/migrate/20211211170003_create_posts.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class CreatePosts < ActiveRecord::Migration[6.1]
+  def change
+    create_table :posts do |t|
+      t.string :string
```
Comment (mke2111) [inline db/migrate/20211211170003_create_posts.rb]: - [x] I guess this should be the title with type of string. Make this change. From `:string` to `:title`

File: db/migrate/20211211170003_create_posts.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+      t.text :text
+      t.integer :comments_counter
+      t.integer :likes_counter
+
+      t.timestamps
+    end
+  end
+end
```
Comment (mke2111) [inline db/migrate/20211211170003_create_posts.rb:12]: - [ ] Let us put this into consideration that a post belongs to a user. So, you need to add a column for the `author_id`

File: db/migrate/20211211170003_create_posts.rb (hunk 1 above)
Comment (OLIPLICHE) [reply db/migrate/20211211170003_create_posts.rb]: done

**Review by AshakaE — APPROVED** [review]
### Hi @OLIPLICHE   🙋‍♀️, 

# STATUS APPROVED ✔
   
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**